sound/soc/sof/ipc4-priv.h

```diff
@@ -17,18 +17,27 @@
 #define SOF_IPC4_OUTBOX_WINDOW_IDX	1
 #define SOF_IPC4_DEBUG_WINDOW_IDX	2
 
+enum sof_ipc4_mtrace_type {
+	SOF_IPC4_MTRACE_NOT_AVAILABLE = 0,
+	SOF_IPC4_MTRACE_INTEL_CAVS2X,
```

Comment:
I still don't get this definition, can we not list all the possible traces today, and later add support for 1.5 and 1.8.
We should describe everything we know, not add a random value just because we tested this on TGL only.

Reply:
I'm aware of those two other variations, but I have seen a third one in some place, but can not connect it to anything.
But if we add them, should we document them as well? They are different than similar to the one we implement here.

Reply:
no, just like we added variants of cAVS architectures, we can add variants of the trace. We don't need to document anything, just be consistent on which machine uses which version of the trace.